tests/ovirt/requests/compute/client_tests.rb

```diff
@@ -0,0 +1,16 @@
+Shindo.tests('Fog::Compute[:ovirt] | client', ['ovirt']) do
+  client_mock = Object.new
+  object_under_test = Fog::Compute::Ovirt::ExceptionWrapper.new(client_mock)
+
+  def client_mock.foo
+    raise OVIRT::OvirtException, 'Test'
+  end
+
+  begin
+    object_under_test.foo
+  rescue Fog::Ovirt::Errors::OvirtClientError => e
+    error = e
+  end
+
+  tests('Has valid backtrace').returns(error.nil?) { false }
```

Comment:
Almost there :smile: 
I think we can skip the backtrace test altogether, instead, can you please test that:

1. Exception properly wraps the text: `error.message == 'Ovirt client returned an error: '`
2. Exception includes the original erorr: `error.orig_exception.is_a?(OVIRT::OvirtException) && error.orig_exception.message == 'Test'`

Reply:
The second test exists, but forgot to commit it (sorry).

I think that your suggestion for the first test is actually much better then what I was thinking in doing :)